src/prommis/solvent_extraction/ree_og_distribution.py

```diff
@@ -144,5 +144,17 @@ def get_material_flow_terms(self, p, j):
                 to_units=units.mol / units.hour,
             )
 
+    def get_material_density_terms(self, p, j):
+        if j == "DEHPA":
+            return units.convert(
```

Comment:
The approach is probably suitable here, where the organic solvent dominates the system.

Reply:
Okay